src/adapter/src/coord/id_bundle.rs

```diff
@@ -26,6 +26,13 @@ pub struct CollectionIdBundle {
 }
 
 impl CollectionIdBundle {
+    pub fn empty() -> Self {
```

Comment:
This can get removed, use default instead.

Reply:
done